rt/mlc/TESTDC1.MLC

```diff
@@ -0,0 +1,461 @@
+         CLC   =C'stuvwxyz',=X'A2A3A4A5A6A7A8A9AA'
+         RT1   CCE
+         CLC   =C"stuvwxyz",=X'737475767778797A7B'
+         RT1   CCE
+         CLC   =C'0123456789',=X'F0F1F2F3F4F5F6F7F8F9'     
+         RT1   CCE
+         CLC   =C"0123456789",=X'30313233343536373839'     
+         RT1   CCE
+* EBCDIC special characters
+       CLC =X'405A7F7B5B6C507D4D5D5C4E6B604B61',=C' !"#$%&&''()*+,-./' 
+         RT1   CCE
+         CLC =X'7A5E4C7E6E6F',=C':;<=>?' 
+         RT1   CCE
+         CLC =X'7C',=C'@'  
+         RT1   CCE
+         CLC =X'E06D',=C'\_'  
+         RT1   CCE
+         CLC =X'4FA1',=C'|~' 
+         RT1   CCE
+         AGO   .TEST4_END
+.TEST4_ASCII ANOP
+         CLC   =C'ABCDEFGHI',=X'414243444546474849'
+         RT1   CCE
+         CLC   =C'JKLMNOPQR',=X'4A4B4C4D4E4F505152'
+         RT1   CCE
+         CLC   =C'STUVWXYZ',=X'535455565758595A5B'
+         RT1   CCE
+         CLC   =C'abcdefghi',=X'616263646566676869'
+         RT1   CCE
+         CLC   =C'jklmnopqr',=X'6A6B6C6D6E6F707172'
+         RT1   CCE
+         CLC   =C'stuvwxyz',=X'737475767778797A7B'
+         RT1   CCE
+         CLC   =C'0123456789',=X'30313233343536373839'     
+         RT1   CCE
+* ASCII special characters
+         CLC =X'20',=C' '
+         RT1   CCE
+         CLC =X'3C282B7C',=C'<(+|' 
+         RT1   CCE
+         CLC =X'26',=C'&&'
+         RT1   CCE
+         CLC =X'21242A293B5E',=C'!$*);^'
+         RT1   CCE
+         CLC =X'2D2F',=C'-/' 
+         RT1   CCE
+         CLC =X'2C255F3E3F',=C',%_>?' 
+         RT1   CCE
+         CLC =X'603A2340273D22',=C'`:#@''="' 
+         RT1   CCE
+         CLC =X'7B',=C'{' 
+         RT1   CCE
+         CLC =X'7D',=C'}' 
+         RT1   CCE
+         CLC =X'7E5B',=C'~[' 
+         RT1   CCE
+         CLC =X'5D',=C']' 
+         RT1   CCE
+.TEST4_END ANOP
+* test && and '' in comments (this), setc, sdt, and models
+&AMP     SETC  '&'  DSH #233 fix setc signle &
+         AIF   ('&AMP' EQ '&').AMPOK  DSH #233
+         RT1   ABORT
+.AMPOK   ANOP
+&SQ      SETC  ''''
+         AIF   ('&SQ' EQ '''').SQOK
+         RT1   ABORT
+.SQOK    ANOP
+         AIF   (C'A' NE C!A!).TEST5_ASCII
+.TEST5_EBCDIC ANOP
+         CLC   =C'&AMP&AMP',=X'50'
+         RT1   CCE
+         CLC   =C'&&',=X'50'
+         RT1   CCE
+         CLI   =C'&&',C'&&'
+         RT1   CCE
+         CLI   =C'&&',C!&&!
+         RT1   CCE
+         CLC   =C'&SQ&SQ',=X'7D'
+         RT1   CCE
+         CLC   =C'''',=X'7D'
+         RT1   CCE
+         CLI   =C'''',C''''
+         RT1   CCE
+         CLI   =C'''',C!''!
+         RT1   CCE
+         CLI   =C'!',C!!!!
+         RT1   CCE
+         CLI   =C'"',C!"!
+         RT1   CCE
+* TEST CA/CE/AD/FD/VD
+         RT1   CCE,=CA'1',=X'31'
+         RT1   CCE,=CE'1',=X'F1'
+         RT1   CCE,=ca'1',=X'31'
+         RT1   CCE,=ce'1',=X'F1'
+         RT1   CCE,=AD(123),=FL8'123'
+         RT1   CCE,=FD'123',=FL8'123'
+TESTSUB1 RT1   CCH,=VD(TESTSUB1),=AL8(TESTSUB1) RPI 1044 DIFF ADDR
```

Comment:
Question only. The ASCII test below uses CVTTOHEX. Why not use it here as well? That way, you do not have to add TESTSUB1.MLC.